pkg/bbgo/interact.go

```diff
@@ -21,6 +21,13 @@ type PositionReader interface {
 	CurrentPosition() *types.Position
 }
 
+type StrategyController interface {
+	SuspendStrategy(ctx context.Context) error
+	ResumeStrategy() error
+	GetStrategyStatus() bool
```

Comment:
We might need to define the strategy statuses

Reply:
done